ingestclient/core/backend.py

```diff
@@ -410,11 +410,11 @@ def join(self, ingest_job_id):
                     raise Exception("After {} attempts, failed to join ingest job: {}".format(maximum_retries, r.text))
 
                 exp_backoff = 100 * 2 ** retries  # in ms
-                pause_for = random.uniform(1, min(30000, exp_backoff)) / 1000
-                print("Join request failed with: {} pausing for {:0.3f} seconds before retrying.".format(r.status_code, pause_for))
+                pause_for = random.uniform(1, min(300000, exp_backoff)) / 1000
```

Comment:
This hits 300,000 after about 12 tries. Maybe we should use a bigger range? Also, the first couple tries will give sleeps of less than 1s. Maybe use `exp_backoff = 100 * 2 ** (retries+4)`?

Reply:
Ok we can do 2**(retries+4) that will be 3.2 secs first round 6.4 secs second 12.4.
I'll raise it from 300,000 to 1,000,000 ms.  Lets say we have 2000 processes hitting at max size we would have 2 threads requesting creds every second. Seems reasonable.

Reply:
Added 400 to exp backoff retries